### ISSUE TITLE
Gelatinous drop-like corneal dystrophy

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9661]
Hello;
searching for clinical signs and symptoms of "ORPHA 98957 Gelatinous drop-like corneal dystrophy"
I found "**foreign body sensation**" as a frequent symptom, however I don't find any similar term in the ontology, I'm rigth? 
Do I have to ask it as a new term?

**Comment by pnrobinson** [comment]
@carolina-f 
I think this would be good as a term underneath
[Corneal dystrophy HP:0001131](https://hpo.jax.org/app/browse/term/HP:0001131)

One problem with the current HPO is that "Corneal dystrophy" is a disease characterized by a certain kind of corneal opacity as well as other manifestations such as photophobia and reduced visual acuity. There does not seem to be a word for the specific corneal manifestation of the corneal dystrophy. I think therefore that we should call it


Gelatinous drop-like dystrophic cornea to specific the phenotypic finding.
ORDO would likely have a disease term called Gelatinous drop-like corneal dystrophy that would be annotated to the HPO terms
Gelatinous drop-like dystrophic cornea
Photophobia
Reduced visual acuity
etc.

Does this make sense?

Here is how I would define the term

Gelatinous drop-like dystrophic cornea
Def: Deposition of amyloid material in the subepithelial space of the cornea
Comment: See Figure 2 in PMID: 23038033

@Sukram

**Comment by carolina-f** [comment]
Hello Robert,
The proposed term Gelatinous drop-like dystrophic cornea seems fine to me.
However, I did not write to ask for a new term for the disease, but only to ask if there was not already a term for the foreign body sensation that they describe among their symptoms.
Not having an answer after a while, I decided to request the new term that was accepted HP:0034804 as Corneal foreign body sensation.

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000712
- label: Gelatinous drop-like dystrophic cornea
- parents: [Corneal dystrophy (HP:0001131)](https://hpo.jax.org/app/browse/term/HP:0001131)
- pmid: PMID:23038033
- def: Deposition of amyloid material in the subepithelial space of the cornea.
- comment: See Figure 2 in PMID:23038033.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9661
- synonyms:
